README.md

```diff
@@ -5,3 +5,40 @@ Notepad++ supports around 80 programming languages for syntax highlighting & fol
 To learn all about User Defined Languages:
 
 https://npp-user-manual.org/docs/user-defined-language-system/
+
+## Using one of these User Defined Languages
+
+For now, you have to manually install a new User Defined Language (UDL).
+
+1. Download the XML file(s) from the [`UDLs`](./UDLs) folder of [this Collection](https://github.com/notepad-plus-plus/userDefinedLanguages).
+2. Import the file(s) either by placing the file(s) in your `userDefineLangs` folder and restarting Notepad++, or by using the User Defined Language dialog box to **Import** your file(s).  More details of what those steps entail can be found in the ["Import a UDL" section](https://npp-user-manual.org/docs/user-defined-language-system/#import-a-udl) of the official documentation.
```

Comment:
Or maybe I should withdraw "Import" feature...

Reply:
Or make the **Import** button just copy the XML file into the folder, rather than merge into the old single-xml file.  If that feature gets changed in the codebase, I'll update the docs accordingly.